python/pyarrow/parquet/core.py

```diff
@@ -1714,11 +1185,13 @@ def _open_dataset_file(dataset, path, meta=None):
 
 
 class ParquetDataset:
-
     __doc__ = """
 Encapsulates details of reading a complete Parquet dataset possibly
 consisting of multiple files and partitions in subdirectories.
 
+Keywords `metadata`, `split_row_groups=True` and `validate_schema=False`
+are not yet supported.
```

Comment:
We could maybe also leave this out? The wording now suggests that we are planning to support them, and I am not sure if that is the case?